notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py

```diff
@@ -245,6 +245,7 @@
     ax1.axes.set_axis_off()
 
     mp = scaled_image.max_proj(Indices.ROUND, Indices.CH, Indices.Z)
```

Comment:
the pattern you took with some of the other files were:

xx_max = xx.max_proj(..)
xx = xx_max._squeezed_numpy(..)

do you want to keep with that pattern here? or alternatively, do what you did here for the other cases?